absl/tests/app_test.py

```diff
@@ -71,7 +71,7 @@ def test_usage(self):
       app.usage()
     self.assertIn(__doc__, mock_stderr.getvalue())
     # Assert that flags are written to stderr.
-    self.assertIn('\n  --[no]helpfull:', mock_stderr.getvalue())
+    self.assertIn('\n  --[no]helpful:', mock_stderr.getvalue())
 
```

Comment:
This is not a typo, this "help" "full" as opposed to "help" "short"

Reply:
I see. Got it.